src/main/java/hu/blackbelt/judo/sdk/CompositeClassLoader.java

```diff
@@ -0,0 +1,93 @@
+package hu.blackbelt.judo.sdk;
+
+/*-
+ * #%L
+ * Structured map proxy
+ * %%
+ * Copyright (C) 2018 - 2022 BlackBelt Technology
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.net.URL;
```

Comment:
Check this is used by generated SDK or just a casual util class which can be removed from here.

Reply:
[80aa3d9](https://github.com/BlackBeltTechnology/judo-sdk-common/pull/69/commits/80aa3d954fccf6d23d4331268e5c53683536583c)